<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitInfo</class>
 <widget class="QWidget" name="UnitInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UnitInfo</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="name">
     <property name="text">
      <string>name</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>HP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="hpBar">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string>%v / %m</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>MP:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="mpBar">
       <property name="maximum">
        <number>300</number>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::BottomToTop</enum>
       </property>
       <property name="format">
        <string>%v / %m</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>等级：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="level">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>level</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>经验：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="expBar">
       <property name="value">
        <number>24</number>
       </property>
       <property name="format">
        <string>%v / %m</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>技能1：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="skill1">
       <property name="text">
        <string>skill1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="useSkill1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>使用</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>技能2：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="skill2">
       <property name="text">
        <string>skill2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="useSkill2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>使用</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>技能3：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="skill3">
       <property name="text">
        <string>skill3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="useSkill3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>使用</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="move">
     <property name="text">
      <string>移动</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
